include/novatel_edie/decoders/common/message_decoder.hpp

```diff
@@ -41,19 +45,23 @@ struct FieldContainer;
 
 #define NOVATEL_TYPES bool, int8_t, int16_t, int32_t, int64_t, uint8_t, uint16_t, uint32_t, uint64_t, float, double, std::string
 #define CONTAINER_TYPES NOVATEL_TYPES, std::vector<FieldContainer>
+using FieldValueVariant = std::variant<CONTAINER_TYPES>;
```

Comment:
I recall we used to have an alias for a vector of fieldcontainers and I removed it because I thought it was a simple structure and more didactic to just write it as the underlying type.